models/app.models.js

```diff
@@ -40,3 +40,20 @@ exports.getArticlesFromDatabase = () => {
     });
   });
 };
+
+exports.getCommentsFromDatabase = (article_id) => {
+  if (article_id && !/^\d+$/.test(article_id))
+    return Promise.reject({ msg: "invalid article id", status: 400 });
```

Comment:
It's unnecessary to do a manual check on the data type of article id. 

If you try to query the database with an article_id that isn't a number - psql will throw an error itself. Then you can catch that and use it - instead of creating your own.